### PR TITLE
Update the basic release checklist for 2.7

### PR DESCRIPTION
**Pull request by maxsokolovsky**
Issue: https://github.com/rancher/rancher/issues/38795

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,12 +105,12 @@ You can also use tools like certmanager's [trust operator](https://cert-manager.
 ## Release
 
 1. Check Kubernetes and Rancher version limits in the annotations of this repo's `charts/Chart.yaml`. Change the supported Kubernetes versions (`kube-version` range) if you have added/removed support for a version in the current range. Change the `rancher-version` range only when making a new major version of the csp-adapter.
-2. Update the [rancher-csp-adapter chart](https://github.com/rancher/charts/blob/dev-v2.6/packages/rancher-csp-adapter/package.yaml):
+2. Update the [rancher-csp-adapter chart](https://github.com/rancher/charts/blob/dev-v2.7/packages/rancher-csp-adapter/package.yaml):
     1. Clone this repo (not a fork).
-    2. Pull the `release/v2.6` branch.
+    2. Pull the `release/v2.7` branch.
     3. Tag the branch with a new version. For example, to release version `1.1.2`, tag with `git tag v1.1.2` (make sure to include the `v`, it's important for the CI).
```
Comment (MbolotSuse) [inline README.md]: Can you update this to have version 2 as the major? I want that to be in the readme somewhere.

**Review by MbolotSuse — CHANGES_REQUESTED** [review]
One minor change on the major version in the example.

File: README.md (hunk 1 above)
Comment (maxsokolovsky) [reply README.md]: For the Git tag example? Sure!

**Review by MbolotSuse — APPROVED** [review]
LGTM

**Pull request merged**